js_modules/dagster-ui/packages/ui-core/src/ui/Filters/useTimeRangeFilter.tsx

```diff
@@ -163,7 +163,7 @@ export function useTimeRangeFilter({
       },
       activeJSX: (
         <ActiveFilterState
-          activeFilterTerm={activeFilterTerm}
+          activeFilterTerm={activeFilterTerm || name}
```

Comment:
I think `activeFilterTerm = 'Timestamp'` wold be preferable as a default over the name of the filter.